docker-registry-sync/reposync/src/reposync/main.py

```diff
@@ -76,7 +76,7 @@ def run_dregsy_task(dregsy_task: Task, results_queue: Queue, debug: bool) -> Non
                 else:
                     current_logs.append(decoded_line)
         results_queue.put((completed_successfully, current_logs, dregsy_task.id))
-    except Exception:
+    except Exception:  # pragma: no cover
```

Comment:
what is this? I know C++ pragmas but in python?? you meant ``pylint: disable=...``?

Reply:
Not it's just a way to instruct `coverage` to ignore some lines which you want to be skipped. In [this project it is configured via tox](https://github.com/ITISFoundation/dockerfiles/blob/339ec1be1863b173063901b4dbf82086896a0593/docker-registry-sync/reposync/tox.ini#L27-L28) otherwise a `.coverage` file is needed to properly set it up.

If the `exclude_lines ` array is not overwritten, the `# pragma: no cover` is the default entry. You can also add other entries, [read more here](https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html).